pkg/controller/importconfig/manifests/klusterlet/klusterlet.yaml

```diff
@@ -7,3 +7,12 @@ spec:
   workImagePullSpec: "{{ .WorkImageName }}"
   clusterName: "{{ .ManagedClusterNamespace }}"
   namespace: "{{ .KlusterletNamespace }}"
+  nodePlacement:
+    nodeSelector:
+    {{- range $key, $value := .NodeSelector }}
```

Comment:
can NodeSelector be nil here?

Reply:
nil is ok here, but NodeSelector must be defined in config.